NSS-Day-Cohort-66/rare-api-golden-droids
# Added update method for Post and it's test

Added a `destroy` method to Post View

Supported routes
`/posts/n` Will return a serialized JSON object

## To Test
1. Pull down `ts-destroy-post` branch
2. Make sure your virtual environment is running
3. In your terminal run the following code:
```
python3 manage.py test tests -v 1
```
4. Be sure you get the following response back
```
Found 15 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
...............
----------------------------------------------------------------------
Ran 15 tests in 0.387s

OK
Destroying test database for alias 'default'...
```
